ddtrace/sampler.py

```diff
@@ -199,6 +200,34 @@ def __init__(
             )
             self.default_sampler = SamplingRule(sample_rate=default_sample_rate)
 
+    def parse_from_env_variable(self):
+        rules = []
+        # DD_TRACE_SAMPLING_RULES = ['{"sample_rate": "1", "service": "b", "name": 0.1}', ....]
+        env_rules = get_env("trace", "sampling_rules")
+        if env_rules is not None:
+            json_rules = []
+            try:
+                json_rules = json.loads(env_rules)
+            except json.decoder.JSONDecodeError:
+                log.debug(
```

Comment:
same here, we probably wait to be much noisier/fail/etc

Reply:
👍